src/js/BookReader.js

```diff
@@ -1503,12 +1502,12 @@ BookReader.prototype.prepareThumbnailView = function() {
   // $$$ keep select enabled for now since disabling it breaks keyboard
   //     nav in FF 3.6 (https://bugs.edge.launchpad.net/bookreader/+bug/544666)
   // utils.disableSelect(this.$('#BRpageview'));
+  const defaultThumbnailWidth = 100;
 
-  this.thumbWidth = this.getThumbnailWidth(this.thumbColumns);
+  const idealReduce = this.getThumbnailWidth(this.thumbColumns);
```

Comment:
Actually, can we put `DEFAULT_THUMBNAIL_WIDTH` inside this function? So it returns the default if it gets a number <= 0?

Reply:
yep